### DISCUSSION TITLE
Connection failed in password + 2FA

### DISCUSSION THREAD

**Discussion by jasperhyp** [Eugeny/tabby#9174] [Q&A]
Hi,

Thank you for the great tool! I am currently ssh-ing to a HPC which requires password + 2-factor authentication. I am able to use the "Interactive" authentication method to access the cluster and would like to save the password so that the connection can be automatically proceeded till the point of 2FA. However, I am not able to use "Password" or "Auto" for this. In the "Auto" mode, I am still prompted to input passwords and 2FA; while in the "Password" mode, it simply says `All configured authentication methods failed`.

Any insights would be much appreciated!

Best,
Yepeng

**Comment by satacata** [comment]
Why don't these morons answer this simple question? Android apps do it easily!

**Reply by Eugeny** [reply]
🤡

**Comment by Eugeny** [comment]
It's because your server isn't actually doing password authentication. It's doing keyboard-interactive authentication where the first prompt says "Password:". Tracked in #1790
